src/assets/scripts/client/navigationLibrary/StandardRoute/RouteSegmentCollection.js

```diff
@@ -137,6 +138,23 @@ export default class RouteSegmentCollection extends BaseCollection {
     }
 
     /**
+     * Returns an array of fix names used by any segment in the collection
+     *
+     * @for RouteSegmentCollection
+     * @method getAllFixNames
+     * @return {array<string>} ['fixname', 'fixname', 'fixname', ...]
+     */
+    getAllFixNames() {
+        let fixNames = [];
+
+        this._items.forEach((routeSegmentModel) => {
```

Comment:
Because with map I'll end up with nested arrays rather than a big pile of fixes. I could `.map()` and then `_flatten`?